tests/unit/conftest.py

```
@@ -88,6 +108,14 @@ def pytest_collection_modifyitems(config, items):
 
     # Filter by automodel marker
     if run_automodel_only:
+        # Validate that nemo_automodel is available
+        try:
+            import nemo_automodel  # noqa: F401
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)